src/api.ts

```diff
@@ -1,7 +1,7 @@
 import { MovieData } from './types/MovieData';
 import { ResponseError } from './types/ReponseError';
 
-const API_URL = 'https://www.omdbapi.com/?apikey=your-key';
+const API_URL = 'http://www.omdbapi.com/?i=tt3896198&apikey=a180f2a6';
```

Comment:
![image](https://github.com/mate-academy/react_movies-list-fetch-movies/assets/91411692/7012f278-9aec-46a9-a956-a3e960918e97)

```suggestion
const API_URL = 'https://www.omdbapi.com/?i=tt3896198&apikey=a180f2a6';
```